.github/workflows/ci.yml

```diff
@@ -148,7 +157,7 @@ jobs:
       - name: Set up GITHUB_TOKEN
         run: |
           echo "GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}" >> $GITHUB_ENV
-      - uses: DeterminateSystems/nix-installer-action@main
+      - uses: DeterminateSystems/nix-installer-action@main # Using the DS Nix installer because it also sets up binfmt
```

Comment:
What would the users be doing to set up `binfmt`?

Reply:
They won't. binfmt is only necessary here because we're emulating other architectures on Linux.